scylladb/README.md

```diff
@@ -0,0 +1,136 @@
+---
+id: scylladb
+title: ScyllaDb
+---
+
+# ScyllaDb
+
+A ScyllaDb storage engine for [Fiber](github.com/gofiber/fiber) using [gocql](github.com/gocql/gocql).
```

Comment:
Is this the right lib? Gocql?

Reply:
According to their docs we should be using this https://github.com/scylladb/gocql

Source: https://opensource.docs.scylladb.com/stable/using-scylla/drivers/cql-drivers/scylla-go-driver.html